telemetry/statsd/src/main/scala/io/buoyant/telemetry/StatsDInitializer.scala

```diff
@@ -0,0 +1,56 @@
+package io.buoyant.telemetry
+
+import com.timgroup.statsd.NonBlockingStatsDClient
+import com.twitter.finagle.Stack
+import com.twitter.finagle.util.DefaultTimer
+import com.twitter.logging.Logger
+import io.buoyant.telemetry.statsd.{StatsDStatsReceiver, StatsDTelemeter}
+import java.util.UUID
+
+class StatsDInitializer extends TelemeterInitializer {
+  type Config = StatsDConfig
+  val configClass = classOf[StatsDConfig]
+  override val configId = "io.l5d.statsd"
+}
+
+private[telemetry] object StatsDConfig {
+  val DefaultPrefix = "linkerd"
+  val DefaultHostname = "127.0.0.1"
+  val DefaultPort = 8125
+  val DefaultGaugePeriodMs = 10000
+
+  val MaxQueueSize = 10000
+  val ProcessId = "l5d-uuid-" + UUID.randomUUID().toString
+}
+
+case class StatsDConfig(
+  prefix: Option[String],
+  hostname: Option[String],
+  port: Option[Int],
+  gaugePeriodMs: Option[Int]
+) extends TelemeterConfig {
+  import StatsDConfig._
+
+  private[this] val log = Logger.get("io.l5d.statsd")
+
+  // prefix format is: "prefix_l5d-uuid-process-id"
+  val statsDPrefix = prefix.getOrElse(DefaultPrefix) + "_" + ProcessId
```

Comment:
That's correct. The UUID is intended to distinguish the same stat coming from different linkerd processes. This can be useful if a single instance of a service is unhealthy. To evaluate the overall health of a service, these stats are intended to be rolled up, ignoring linkerd UUID.

Does this make sense in the StatsD world? In testing a StatsD+Graphite setup, I did not see another way to distinguish stats coming from multiple linkerd processes. As a counter-example, Prometheus identifies metric sources by IP address by default.